charts/rpe-send-letter-service/Chart.yaml

```diff
@@ -1,12 +1,12 @@
 name: rpe-send-letter-service
 apiVersion: v2
 home: https://github.com/hmcts/send-letter-service
-version: 0.2.0
+version: 0.2.1
 description: HMCTS Send letter service
 maintainers:
   - name: HMCTS BSP Team
     email: bspteam@hmcts.net
 dependencies:
   - name: java
-    version: ~3.0.0
```

Comment:
I _think_ we can actually leave this change for another PR to save complexity. I don't think this has an impact on the azurerm provider

Reply:
Ok, I will leave this change